src/service/error.svc.ts

```diff
@@ -0,0 +1,32 @@
+export const isError = (error: unknown): error is Error => {
```

Comment:
```suggestion
export const isError = (error: unknown): error is Error => {
```

do we really need it 😄

Reply:
yeah `isError` this is an alias for instanceof Error, its not very useful